### PR TITLE
Add test for test suite action util addTestToSuite

### PR DESCRIPTION
**Pull request by reubenae**
Thank you for contributing this pull request!

Please make the PR against the `master` branch, add a description of what's changing, and link any relevant issues or PRs.

Part of https://github.com/intuit/ReplayWeb/issues/5

## What's changing

Added a test for the suites action addTestToSuite function

...

## What else might be impacted? 

...

## Checklist

[x] Unit tests (updated and/or added)
[x] Documentation (function/class docs, comments, etc.)

Published PR with canary version: `1.0.1-canary.23.267.0`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,11 +34,19 @@
...
+    "@wdio/local-runner": "^5.0.0",
+    "@wdio/mocha-framework": "^5.0.0",
+    "@wdio/selenium-standalone-service": "^5.0.0",
+    "@wdio/spec-reporter": "^5.0.0",
     "auto": "^7.8.0",
     "babel-eslint": "^10.0.3",
     "babel-jest": "^24.9.0",
     "babel-plugin-add-module-exports": "^1.0.2",
     "babel-plugin-rewire": "^1.2.0",
+    "chai": "^4.0",
```
Comment (Celeo) [inline package.json]: Is this coming from some other package or being added as a new dependency?

File: package.json [hunk 2]
```diff
@@ -76,5 +85,8 @@
       "eslint --fix",
       "git add"
     ]
+  },
+  "dependencies": {
+    "@replayweb/testrunner": "1.0.0"
```
Comment (Celeo) [inline package.json]: Does this need to be here to enable some functionality?

File: package.json (hunk 1 above)
Comment (reubenae) [reply package.json]: My bad - these shouldn't be here. Let me clean up the PR.

**Comment by reubenae** [comment]
@Celeo thanks for reviewing. I wasn't sure if I should be doing some cleanup in my tests (like resetting my mocks?)

**Comment by Celeo** [comment]
Sure, reset mocks if you need them for test-to-test assertions, i.e. asserting that a mock was called twice in test *A*, once in test *B*, never in test *C*, etc. If you don't need that functionality, then I don't think they need to be reset.

**Review by Celeo — APPROVED** [review]

**Comment by reubenae** [comment]
Sounds good, I think this PR is ready to merge then! I'll add some more tests over the coming days.

**Comment by Celeo** [comment]
Sounds good, thanks for the contribution!

**Comment by hipstersmoothie** [comment]
:rocket: PR was released in v1.0.1 :rocket:

**Pull request merged**